### DISCUSSION TITLE
跨國人工智慧（AI ）數位轉型及數據行銷整體解決方案供應商愛卡拉（iKala）協助企業導入AI，提出 「DAA飛輪架構」

### DISCUSSION THREAD

**Discussion by skyling0299** [1101-GS3073-C/Python-Community-GS3073-C#547] [Q&A]
跨國人工智慧（AI ）數位轉型及數據行銷整體解決方案供應商愛卡拉（iKala）協助企業導入AI，提出 「DAA飛輪架構」

查詢網路資料後發現這間公司似乎並沒有太大的知名度，根據PTT徵才文章，iKala 是一間跨國的 AI 公司，使命是「AI 賦能」，讓我們的客戶能夠以AI 為核心來達成事業轉型、加速、及創造新商業模式的目標。iKala 提供以 AI 驅動的數位轉型及數據行銷整體解決方案，旗下有 iKala Cloud 及iKala Commerce 等雲端及新零售服務與產品，服務超過 400 家企業及15,000 家廣告主及品牌主，其中包含多家財星 500 大企業，服務範圍跨越7 個國家（新加坡、台灣、日本、香港、越南、泰國、菲律賓）及超過12 種產業。iKala 以顧客為念，不斷提升服務品質及投資創新研發，致力為顧客創造更多的價值。iKala 同時是 Google 及 Facebook 的合作夥伴。

想討論關於飛輪架構以及關於這間公司到底是在進行哪些方面的AI引進

資料來源: https://ec.ltn.com.tw/article/breakingnews/4145350

**Comment by williamyen0623** [comment]
iKala這種作法感覺可以幫助到各種不同的企業不會在科技蓬勃發展的時代被淘汰。
